cmd/admxgen/defs/mount.yaml

```diff
@@ -17,3 +17,25 @@
   type: "mount"
   meta:
     strategy: "append"
+
+- key: "/system-mounts"
+  displayname: "System mounts"
+  explaintext: |
+    Define network shares that will be mounted for the system.
+    If more shares are defined higher in other GPO, the entries listed here will be appended to the list and duplicates will be removed.
```

Comment:
`higher in the GPO hierarchy`? (if we are not consistent with other descriptions, we should)